test/report.js

```diff
@@ -48,8 +48,6 @@ describe('Concur Report Tests', function() {
     });
   });
 
-  var reportId;
```

Comment:
The test will fail without this. Perhaps it needs a refactor. This gives the ability to use a report ID throughout the tests.